### PR TITLE
Adding gpu type

### PR DESCRIPTION
**Pull request by moshe-kabala**

### CODE REVIEW

**Comment by moshe-kabala** [comment]
I added a function called SetLabel to define an index label on a string, but I do not use it. I think it might be useful in the feature. If you do not agree let me know and I will remove it

File: pkg/prometheus/helpers.go [hunk 1]
```diff
@@ -20,6 +21,29 @@ func SetFloatFromFirstMetric(num *float64, m MetricResultsByQueryName, key strin
 	return nil
 }
 
+func SetLabel(str *string, label string ,m MetricResultsByQueryName, key string) error {
```
Comment (yodarshafrir1) [inline pkg/prometheus/helpers.go:24]: why do we need this?

File: pkg/nodes/node_info.go [hunk 2]
```diff
@@ -84,6 +84,7 @@ func (ni *NodeInfo) GetResourcesStatus() types.NodeResourcesStatus {
 	nodeResStatus.Allocated = podResStatus.Requested
 	nodeResStatus.Allocated.GPUs = podResStatus.Allocated.GPUs // needed to count fractions as well
 	nodeResStatus.Limited = podResStatus.Limited
+	nodeResStatus.GpuType = ni.Node.Labels["nvidia.com/gpu.product"]
```
Comment (yodarshafrir1) [inline pkg/nodes/node_info.go:87]: if label not found please set a proper message

**Review by yodarshafrir1 — CHANGES_REQUESTED** [review]
please see my notes and attach a screenshot

File: pkg/nodes/node_info.go (hunk 2 above)
Comment (moshe-kabala) [reply pkg/nodes/node_info.go:87]: in the field itself? which text? (Not found or - )
by the there is a case of not a GPU node

File: pkg/helpers/node_resources_status_helpers.go [hunk 3]
```diff
@@ -39,6 +39,7 @@ func (c *NodeResourcesStatusConvertor) ToGpus() *types.NodeGPUResource {
 		Usage:       nrs.Usage.GPUs / 100 * float64(capacity),
 		Utilization: nrs.Usage.GPUs,
 		InUse:       nrs.GPUsInUse,
+		Type: 		 nrs.GpuType,
```
Comment (yodarshafrir1) [inline pkg/helpers/node_resources_status_helpers.go:42]: please rename to `GpuType`

File: pkg/types/node.go [hunk 4]
```diff
@@ -16,6 +16,7 @@ type NodeCPUResource struct {
 }
 
 type NodeGPUResource struct {
+	Type        string  `title:"TYPE" def:"-"`
```
Comment (yodarshafrir1) [inline pkg/types/node.go:19]: Please rename to `GpuType`

**Review by yodarshafrir1 — CHANGES_REQUESTED** [review]

**Review by yodarshafrir1 — APPROVED** [review]

**Pull request merged**